src/controller/model/device.ts

```diff
@@ -843,6 +843,18 @@ export class Device extends Entity<ControllerEventMap> {
             MOT003: {}, // https://github.com/Koenkk/zigbee2mqtt/issues/12471
             "C-ZB-SEDC": {}, //candeo device that doesn't follow IAS enrollment process correctly and therefore fails to complete interview
             "C-ZB-SEMO": {}, //candeo device that doesn't follow IAS enrollment process correctly and therefore fails to complete interview
+            "ZG-204ZM": {}, //hobeian device that doesn't follow IAS enrollment process correctly and therefore fails to complete interview
```

Comment:
I want to go for a more generic solution to match on the manufacturerID, could you provide the data/database.db entry o fthis device?